File: flight_busiest_traveling_period/v1/get/Node SDK/flight_busiest_traveling_period.js
const Amadeus = require("amadeus");

const amadeus = new Amadeus({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // What were the busiest months for Madrid in 2017?
    const response =
      await amadeus.travel.analytics.airTraffic.busiestPeriod.get({
        cityCode: "MAD",
        period: "2017",
        direction: Amadeus.direction.arriving,
      });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();

File: flight_busiest_traveling_period/v1/get/Python SDK/flight_busiest_traveling_period.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    '''
    What were the busiest months for Madrid in 2022?
    '''
    response = amadeus.travel.analytics.air_traffic.busiest_period.get(
        cityCode='MAD', period='2022', direction='ARRIVING')
    print(response.data)
except ResponseError as error:
    raise error
